packages/sysfsutils.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Sysfsutils < Package
+  description 'Libsysfs provides a stable programming interface to sysfs and eases querying system devices and their attributes.'
```

Comment:
Shouldn't this description mention sysfsutils instead of libsysfs?

Reply:
If you don't mind the description being slightly long, I could do that.

Reply:
Done.